meteor/lib/Settings.ts

```diff
@@ -76,7 +76,7 @@ const DEFAULT_SETTINGS = Object.freeze<ISettings>({
 	enableKeyboardPreview: true,
 	keyboardMapLayout: KeyboardLayouts.Names.STANDARD_102_TKL,
 	poisonKey: 'Escape',
-	installationName: 'tv2',
+	customizationClassName: 'tv2',
```

Comment:
To minimize conflicts, this should be set to `undefined` (or not defined) here and overridden in our installations using core-settings.json as we do with other settings.

Reply:
I agree that from a broader Sofie perspective then 'tv2' isn't default.

My main issue/concern is, that what we set in core-settings.json isn't run on our local machines. So if we want our styling when running Sofie local, we would have to add it manually here (and I personnally would like to run the same 'skin' of Sofie as our installations).

Maybe I'm missing something that would allows us to use core-settings.json locally. (Would be awesome if we could :) )

Sidenote: The argument: 'Want it so that it works locally' isn't good, so I'm upon for suggestions.

Reply:
You can provide the file during development. but it might be good to somehow incorporate that into our scripts so that we don't always have to remember about adding that parameter. https://nrkno.github.io/sofie-core/docs/user-guide/configuration/sofie-core-settings#settings-file

Reply:
I agree that we should either remove the entry from the default settings or make it undefined.
Then we can either manually supply the configuration file or make it part of our scripts as @ianshade suggests.

Reply:
I have removed it from the `DEFAULT_SETTINGS` and updated our ansible-prod, -staging and -dev repos with the variable